### ISSUE TITLE
domain is down

### ISSUE THREAD

**Issue by galenthurber** [mbusb/multibootusb#536]
expired domain

**Comment by robotshateme** [comment]
Project is abandoned at this time it would seem. Redundant now with Ventoy being actively updated.

**Comment by mil2house** [comment]
Hmm, but Ventoy is not Multi... :(

**Comment by PonasKovas** [comment]
If that is that case I think it would be sensible to provide this information in the readme and link to alternatives.

**Comment by fourlightyears** [comment]
> If that is that case I think it would be sensible to provide this information in the readme and link to alternatives.

Yes, at least remove the link, because visitors to this project can easily be fooled by malicious actors providing binaries purportedly made out of this repo.

**Comment by Pysis868** [comment]
https://www.ventoy.net/en/plugin_autoinstall.html seems not many OSes supported either..

**Comment by spiralofhope** [comment]
> https://www.ventoy.net/en/plugin_autoinstall.html seems not many OSes supported either..

That's not for actual Ventoy, that's for unattended installations of an ISO..

Your comment also has nothing to do with this topic.

**Comment by Pysis868** [comment]
"Bootable USB Solution".  Sure it does, when someone mentioned the alternative tool already and I'm just trying to find more information that would help booting OSes.

Can you just add more info if I really am missing something here instead of trying to gatekeep an issue for a dead repo/site/tool?

**Comment by PonasKovas** [comment]
@Pysis868 if you have questions about Ventoy you are free to use their forum, this issue is not related at all.

**Comment by Pysis868** [comment]
Only continued as the sentiment showed this project was not sufficient for some, and the mention of it was in this issue, which is how I first learned of it.

As long as I have the tool confirmed, I am sure to use this issue being curious about this tool's site being gone and needing the project update to remove or change that link still.

**Comment by spiralofhope** [comment]
> Can you just add more info if I really am missing something here instead of trying to gatekeep an issue for a dead repo/site/tool?

topic:  "domain is down"

People like myself are watching this topic to see if this issue is corrected or if a new domain is registered.

**Comment by AmyMoriyama** [comment]
"Project is abandoned at this time it would seem. Redundant now with Ventoy being actively updated."

Ventoy doesn't work for the Linux versions I want to use, Multibootusb does work. So, multibootusb should continue being developed.

**Comment by spiralofhope** [comment]
At the risk of the hypocrisy for being off-topic..

> @AmyMoriyama 
> Ventoy doesn't work for the Linux versions I want to use, Multibootusb does work. So, multibootusb should continue being developed.

Would you consider [opening a ticket with Ventoy](https://github.com/ventoy/Ventoy/issues/new/choose) for this?  (and tag me in it or link it here)  I do have multbootusb in my toolkit still, and you've given me a good reason to keep it there, but I'm interested in Ventoy's continued improvement.

I know the author is probably going to suggest using the cli version as I saw on [the ticket for pop os support](https://github.com/ventoy/Ventoy/issues/1595):

https://github.com/ventoy/Ventoy/actions/runs/2230495703
https://www.ventoy.net/en/doc_github_ci.html

**Comment by AmyMoriyama** [comment]
> Would you consider [opening a ticket with Ventoy](https://github.com/ventoy/Ventoy/issues/new/choose) for this? (and tag me in it or link it here) I do have multbootusb in my toolkit still, and you've given me a good reason to keep it there, but I'm interested in Ventoy's continued improvement.

The distros I want to use are pre-UEFI and ventoy doesn't like that and it appears the devs either don't want to "fix it" to allow older distros to be booted or can't fix it.

One example are these:
https://archive.org/details/Puppy_Linux_Legacy_Builds

**Comment by mguzmann** [comment]
Domain is still down, and spamming amazon links now. A maintainer should really remove it/change it/fix it.

**Comment by now0clock** [comment]
I found the User Guide on the Wayback Machine from April 9, 2018. This is linked in the README.md in this repo. The archived page was made 6 days after the last update of the README, that is almost 5 & 1/2 years ago!

https://web.archive.org/web/20180409132411/http://multibootusb.org/page_guide/

It looks ok, but the lack of maintenance has turned me off. You can fork this repo, or just use **Ventoy** like others have mentioned. I highly recommend it. https://ventoy.net/


> The distros I want to use are pre-UEFI and ventoy doesn't like that and it appears the devs either don't want to "fix it" to allow older distros to be booted or can't fix it.
> 
> One example are these: https://archive.org/details/Puppy_Linux_Legacy_Builds

 @AmyMoriyama ,

I have a 64GB USB thumb drive with Ventoy and about 12+ distros/tools on it. It boots and runs fine in my Dell laptop from 2012, with oldschool legacy BIOS. It also boots and runs on my newer UEFI laptop. I found an article on their site that might help,..

https://www.ventoy.net/en/doc_legacy_limit.html

Also, while this is a great tool to Live boot Linux distros, you really just gotta uninstall MS Windows and actually install a Real operating system. One that has a Linux kernel. In your case that might be a good idea, would be to just install Puppy Linux on your PC.
